File: src/targets/objc/nsurlsession/fixtures/multipart-form-data-no-params.m
#import <Foundation/Foundation.h>

NSDictionary *headers = @{ @"Content-Type": @"multipart/form-data" };

<<<<<<< HEAD
NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"https://httpbin.org/anything"]
=======
NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"http://mockbin.com/har"]
>>>>>>> upstream/master
                                                       cachePolicy:NSURLRequestUseProtocolCachePolicy
                                                   timeoutInterval:10.0];
[request setHTTPMethod:@"POST"];
[request setAllHTTPHeaderFields:headers];

NSURLSession *session = [NSURLSession sharedSession];
NSURLSessionDataTask *dataTask = [session dataTaskWithRequest:request
                                            completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
                                                if (error) {
                                                    NSLog(@"%@", error);
                                                } else {
                                                    NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
                                                    NSLog(@"%@", httpResponse);
                                                }
                                            }];
[dataTask resume];

File: src/targets/swift/nsurlsession/fixtures/multipart-form-data-no-params.swift
import Foundation

let headers = ["Content-Type": "multipart/form-data"]

<<<<<<< HEAD
let request = NSMutableURLRequest(url: NSURL(string: "https://httpbin.org/anything")! as URL,
=======
let request = NSMutableURLRequest(url: NSURL(string: "http://mockbin.com/har")! as URL,
>>>>>>> upstream/master
                                        cachePolicy: .useProtocolCachePolicy,
                                    timeoutInterval: 10.0)
request.httpMethod = "POST"
request.allHTTPHeaderFields = headers

let session = URLSession.shared
let dataTask = session.dataTask(with: request as URLRequest, completionHandler: { (data, response, error) -> Void in
  if (error != nil) {
    print(error as Any)
  } else {
    let httpResponse = response as? HTTPURLResponse
    print(httpResponse)
  }
})

dataTask.resume()